ill-fated-ducks/Bangazon-Site
# Add product to the cart

## Status
Choose:
**READY**


## Description
I created Buy Product method in Order controller, it will create new order if there is no order for that user, and post to the database

## Todos

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.
1. git fetch
2. git checkout az-add
3. click on  product details link and add product to the cart
4. check your database

## Impacted Areas in Application
List general components of the application that this PR will affect:
Order controller
Product/Details view

## Link to Feature Ticket
#13 
@ill-fated-ducks
